helpers/test-data-generator.js

```diff
@@ -81,6 +85,12 @@ module.exports = {
     await colony.proposeTaskChange(txData, 0, MANAGER_ROLE);
     transactionId = await colony.getTransactionCount.call();
     await colony.approveTaskChange(transactionId, WORKER_ROLE, { from: worker });
+
+    txData = await colony.contract.setTaskPayout.getData(taskId, EVALUATOR_ROLE, tokenAddress, evaluatorPayout.toString());
+    await colony.proposeTaskChange(txData, 0, MANAGER_ROLE);
+    transactionId = await colony.getTransactionCount.call();
+    await colony.approveTaskChange(transactionId, WORKER_ROLE, { from: worker });
```

Comment:
Note that `setTaskPayout` transaction reviewers are currently the task Manager and Worker. However when setting the Evaluator the reviewers logically have to be the Manager and Evaluator. With the current `ColonyTransactionReviewer` design this is not possible. So we have to either:
a) Split the `setTaskPayout` in two functions, e.g. `setTaskWorkerPayout` and `setTaskEvaluatorPayout`. The disadvantage here is that for additional roles we may want to support in future, i.e. in addition to manager, evaluator and worker, we'll have to keep introducing new functions so we lose 1 level of abstraction.

b) We refactor the `ColonyTransactionReviewer` to support this model. 

I haven't explored option b) so feel free to experiment. Overall I'd prefer option a) for simplicity but I want to hear what @area thinks about it.

Reply:
For now i created separate functions for every role. I will try to find better solution after this PR is merged, because these changes are required for #154.